src/modules/feedback/components/FeedbackModal.js

```diff
@@ -57,34 +57,44 @@ type PropsType = {|
   theme: ThemeType
 |}
 
+type SendingStatusType = 'IDLE' | 'SUCCESS' | 'ERROR'
+
 type StateType = {|
-  feedbackSent: boolean
+  feedbackSent: SendingStatusType
 |}
 
 export class FeedbackModal extends React.Component<PropsType, StateType> {
-  state = { feedbackSent: false }
+  state = { feedbackSent: 'IDLE' }
 
-  handleSubmit = () => this.setState({ feedbackSent: true })
+  handleSubmit = (value: SendingStatusType) => {
+    this.setState({ feedbackSent: value })
+  }
 
   handleOverlayClick = () => {
-    this.setState({ feedbackSent: false })
+    this.setState({ feedbackSent: 'IDLE' })
     this.props.closeFeedbackModal()
   }
 
   render () {
     const { theme, feedbackStatus, ...otherProps } = this.props
     const { feedbackSent } = this.state
 
+    let boxContainer
+    if (['IDLE', 'ERROR'].includes(feedbackSent)) {
+      boxContainer = <FeedbackBoxContainer isPositiveRatingSelected={feedbackStatus === POSITIVE_RATING}
+                                   {...otherProps}
+                                   onSubmit={this.handleSubmit}
+                                   feedbackSent={feedbackSent}
+                                   theme={theme} />
+    } else if (['SUCCESS'].includes(feedbackSent)) {
```

Comment:
In fact, no need for an `else if` at all since all possible values  are `IDLE, ERROR, SUCCESS` and you've already handled both of them above. So an `else` is enough.

Reply:
ok.